### PR TITLE
Improve prompts

### PR DESCRIPTION
**Pull request by aantron**

### CODE REVIEW

File: spin [hunk 1]
```diff
@@ -3,10 +3,10 @@
 (description "Dream web application")
 
 (config project_name
-  (input (prompt "Project name")))
+  (input (prompt "Project name:")))
```
Comment (tmattio) [inline spin:6]: I find prompts without colons aesthetically more pleasant. It also follows the GitHub CLI, which is known for having a nice UI:

image

Comment (aantron) [reply spin:6]: The aesthetics of GitHub are not due to this, though. This kind of prompt is very ugly when a terminal or font does not support boldface, as in my case, and it harms legibility considerably. It's not reasonable to rely on only colors, given the wide variety of terminals out there. I have personally used monochrome meme terminals to reduce distractions sometimes. A prompt relying on boldface only is bad for that.

GitHub makes a lot of questionable choices. I suggest looking at git instead. Colons are completely standard, and you'll find them in display in git (I don't have an example of a prompt ready):

```
commit bc65a69d61e2f7f6798ec3a177972097706320dc (HEAD -> prompts, fork/prompts)
Author: Anton Bachin <antonbachin@yahoo.com>
Date:   Tue Jun 29 13:21:25 2021 +0300
```

Comment (aantron) [reply spin:6]: Снимок

Here you can also see that the prompts that end with `?` are much more legible than the ones that don't end with anything. And the two kinds of prompts, question and non-question, are visually inconsistent. Colons are what allows you to introduce consistency into how prompts are terminated.

Comment (tmattio) [reply spin:6]: That makes sense, I didn't think of monochrome terminals. I'll merge it now, but if you have the occasion, I'd be interested in a screenshot to see how the prompt looks like for you.

Comment (tmattio) [reply spin:6]: Our comments crossed, thanks a lot for the screenshot 🙂

Comment (aantron) [reply spin:6]: Besides my current terminal, I've also previously used Cathode on macOS, when I wanted a nice looking terminal but with zero distractions, not even colors. I wasn't using the configuration in the image below, but here is a typical monochrome configuration:

cathode-screenshot

Obviously that's more meme than important, but still :)

And I wouldn't be surprised if lack of colons and reliance on boldface/colors has some accessibility implications, but I'm not an expert :) Anyway, thanks!

**Pull request merged**